cmd/sync/runner.go

```diff
@@ -152,35 +152,33 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 				continue
 			}
 
-			if !tagExists {
```

Comment:
I didn't get why this is removed

Reply:
https://github.com/giantswarm/crsync/pull/20/files#diff-7dda0647dafafce63449dfa490b21807R152

Reply:
:facepalm: